shell/platform/embedder/embedder.h

```diff
@@ -1402,6 +1402,18 @@ typedef void (*FlutterUpdateSemanticsCallback2)(
     const FlutterSemanticsUpdate2* /* semantics update */,
     void* /* user data*/);
 
+/// An update to whether a message channel has a listener set or not.
+typedef struct {
+  /// The name of the channel.
+  const char* channel;
+  /// True if a listener has been set, false if one has been cleared.
+  bool listening;
+} FlutterChannelUpdate;
```

Comment:
Are we 100% certain that we'll never  want to add more members to this struct? If not, add a `size_t struct_size` member as the first field in the struct and use safe dereference macros when reading from it.

Reply:
Oh derp I completely forgot this 🤦. Yup let's add a `struct_size` to be safe

Reply:
By this, do you mean the `SAFE_ACCESS` macro? Do we currently use this at all in the `FlutterWindowsEngine`? It does not appear so, as far as I can tell.